src/main/java/core/basesyntax/strategy/StorageService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface StorageService {
```

Comment:
```suggestion
public interface OperationService {
```
and accordingly all impl